addons/advanced_fatigue/script_component.hpp

```diff
@@ -14,6 +14,7 @@
     #define DEBUG_SETTINGS DEBUG_SETTINGS_ADVANCED_FATIGUE
 #endif
 
+#include "\a3\ui_f\hpp\defineCommonGrids.inc"
```

Comment:
Should be below macros include.

Reply:
Copied from `medical_feedback` ;P
https://github.com/acemod/ACE3/blob/master/addons/medical_feedback/script_component.hpp#L17